packages/core/content-manager/admin/src/index.ts

```diff
@@ -44,6 +44,38 @@ export default {
     });
 
     app.registerPlugin(cm.config);
+
+    // Register homepage widgets
+    app.widgets.register([
+      {
+        icon: Pencil,
+        title: {
+          id: `${PLUGIN_ID}.widget.last-edited.title`,
+          defaultMessage: 'Last edited entries',
+        },
+        component: async () => {
+          const { LastEditedWidget } = await import('./components/Widgets');
+          return LastEditedWidget;
```

Comment:
I think we can just expect the component to be returned (not {default}), I updated the type but lmk if it doesn't make sense  (see the registration below). I figure if it's a default export you just do Compo.default 🤷‍♂️ is that confusing?

Reply:
I think I like that as we've moved more towards named exports, at least internally. It creates an inconsistency with custom fields but probably no big deal